src/test/java/pl/pszczolkowski/mustdo/web/restapi/board/BoardAddEndpointTest.java

```diff
@@ -0,0 +1,69 @@
+package pl.pszczolkowski.mustdo.web.restapi.board;
+
+import javax.transaction.Transactional;
+
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.transaction.TransactionConfiguration;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.web.context.WebApplicationContext;
+
+import com.tngtech.jgiven.annotation.ProvidedScenarioState;
+import com.tngtech.jgiven.junit.ScenarioTest;
+
+import pl.pszczolkowski.mustdo.Application;
+import pl.pszczolkowski.mustdo.domain.board.bo.BoardBO;
+import pl.pszczolkowski.mustdo.domain.board.finder.BoardSnapshotFinder;
+import pl.pszczolkowski.mustdo.domain.board.repository.BoardRepository;
+import pl.pszczolkowski.mustdo.web.restapi.board.steps.GivenBoardAddEndpoint;
+import pl.pszczolkowski.mustdo.web.restapi.board.steps.ThenBoardAddEndpoint;
+import pl.pszczolkowski.mustdo.web.restapi.board.steps.WhenBoardAddEndpoint;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = Application.class)
+@WebAppConfiguration
+@Transactional
+@TransactionConfiguration(defaultRollback = true)
+public class BoardAddEndpointTest extends ScenarioTest<GivenBoardAddEndpoint, WhenBoardAddEndpoint, ThenBoardAddEndpoint>{
```

Comment:
Notice that WhenStage will be always the same for all REST API tests - it has just one method "request_is_sent". So name it like "RestApiWhenStage" and move to util package, so it could be shared among all Rest Api tests